lang/gcc14/Portfile

```diff
@@ -89,9 +99,14 @@ platform darwin {
 }
 
 set gcc_configure_langs {c c++ objc obj-c++ lto fortran}
-if {${subport} eq ${name} && ${build_arch} ne "i386"} {
+if {${subport} eq ${name} && ${build_arch} ne "i386" && ${build_arch} ne "ppc" && ${build_arch} ne "ppc64"} {
```

Comment:
This is not a ppc problem, this is 10.4 problem. Correct condition is `${configure.build_arch} ne "i386" && ${os.major} > 8`.

Reply:
Done, thank you